ethereum/test/TablelandRigs.ts

```diff
@@ -1781,10 +1801,6 @@ describe("Rigs", function () {
       it("Should block contract non-owner", async function () {
         const _rigs = rigs.connect(accounts[2]);
 
-        await expect(
-          _rigs.initPilots(ethers.constants.AddressZero)
-        ).to.be.rejectedWith("Ownable: caller is not the owner");
```

Comment:
Nice, guessing this is a no-op

Reply:
Yeah, I removed it because it is irrelevant to this test case, probably because the test case was copy/pasted. It is covered in another test case!